geode-logging/src/test/resources/expected-pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <!--
```

Comment:
Not sure if matters whether the apache banner is at the very top of the file or not, but just noticed that it isn't in this file

Reply:
seems to be following the convention for these type files